README.md

```diff
@@ -80,8 +80,7 @@ And testing gems like:
 
 Suspenders also comes with:
 
-* The [`./bin/setup`][setup] convention for new developer setup
-* The `./bin/deploy` convention for deploying to Heroku
```

Comment:
I removed this because it's not in line with our convention of using git branches that mirror the names of deployment environments and only deploying from those... I figure we can add something like this back in later if we want to, but for now we're pretty good with CI auto-deploying staging branch anyhow.